debugger.hpp

```diff
@@ -69,7 +69,7 @@ inline void debug_ast(AST_Node* node, string ind = "", Env* env = 0)
         case Complex_Selector::ADJACENT_TO: cerr << "{+}"; break;
```

Comment:
Looks like your using `debug_ast` too :wink: I hope you like it as much as I do :smile: It actually decreased my time hunting bugs dramatically! I also thought about creating a nicer `debug_env` function, but I was able to use existing one (altough the output could be better there).